core/src/main/javascript/app/pages/TimeSeriesExecutions.js

```diff
@@ -137,17 +141,80 @@ class TimeSeriesExecutions extends React.Component {
     let { classes, job, start, end, envCritical } = this.props;
     let { data, sort } = this.state;
 
-    let sortedData = _.sortBy(data, e => {
-      switch (sort.column) {
-        case "backfill":
-          return e.context.backfill;
-        default:
-          return e.endTime;
+    const sortedJobExecutions = jobExecutions => {
+      let result = _.sortBy(jobExecutions, e => {
+        switch (sort.column) {
+          case "backfill":
+            return e.context.backfill;
+          default:
+            return e.endTime;
+        }
+      });
+      if (sort.order == "desc") {
+        result = _.reverse(result);
       }
-    });
-    if (sort.order == "desc") {
-      sortedData = _.reverse(sortedData);
-    }
+      return result;
+    };
+
+    const blockedByRow = (executions: Array<ExecutionLog>) => {
+      const groupedParentExecutions = (executions: Array<ExecutionLog>) =>
+        _(executions)
+          .countBy(e => e.status)
+          .entries()
+          .sortBy(([status, _]) => status)
+          .map(([status, count]) => (
+            <a
+              className={classes.openIcon}
+              onClick={() => {
+                this.setState({
+                  blockedBySelectedStatus: this.state.blockedBySelectedStatus ==
+                    status
+                    ? undefined
+                    : status
+                });
+              }}
+              style={{ cursor: "pointer" }}
+            >
+              <JobStatus
+                key={status}
+                status={status}
+                labelFormatter={s => `${count} - ${s}`}
+              />
+            </a>
+          ))
+          .value();
+      const blockedByExpandedRow = (
+        executions: Array<ExecutionLog>,
+        status: string
+      ) => 
+        _(executions)
+          .filter(e => e.status === status)
+          .sortBy('id')
+          .sortBy('job')
+          .map(e => (
+            <Link href={`/executions/${e.id}`}>
```

Comment:
There is a link on the TODO items, but the link goes nowhere (of course because there is no linked execution yet. Can we link to the period instead? for example /timeseries/executions/hello3/2017-10-16T04Z_2017-10-16T05Z)

Reply:
Fixed !